website/docs/topics/feature-flags/evaluate-flags-close-to-user.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 3. Evaluate flags as close to the user as possible. Reduce latency.
+---
+
+Feature flags should be evaluated as close to your users as possible, and the evaluation should always happen server side as discussed in Principle 3. In addition to security and privacy benefits, performing evaluation as close as possible to your users has multiple benefits:
```

Comment:
I think this might be a typo

```suggestion
Feature flags should be evaluated as close to your users as possible, and the evaluation should always happen server side as discussed in Principle 2. In addition to security and privacy benefits, performing evaluation as close as possible to your users has multiple benefits:
```